kiwi/boot/functions.sh

```diff
@@ -5525,18 +5540,28 @@ function waitForStorageDevice {
     # which could be a whole disk or a partition.
     # the function will wait until the size of the
     # storage device could be obtained or the check
-    # counter equals 4
+    # counter equals 30. For USB devices it waits
+    # only until check counter equals 2.
     # ----
     local IFS=$IFS_ORIG
     local device=$1
     local check=0
+    local transport=$(getDeviceTransportType $device)
+    case "$transport" in
+	"usb")
+	    limit=2
+	;;
+	*)
+	    limit=30
+	;;
```

Comment:
nit picking sorry, could you move the comment information from the function body into the corresponding case blocks so that we know limit=2 means a 4 sec wait period, limit=30 means a 60sec wait period